public/docs/ts/latest/guide/architecture.jade

```diff
@@ -92,21 +92,19 @@ figure
 
```

Comment:
Having " - " as a separator in these items gives me pause, but I suppose it makes sense here since you've already used a colon (": ") to introduce the items.